src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_DeleteProfileScreen.tsx

```diff
@@ -33,9 +28,24 @@ export const Profile_DeleteProfileScreen = ({
     filterActiveOrCanBeUsedFareContracts(fareContracts, serverNow).length > 0;
 
   const [deleteError, setDeleteError] = useState<boolean>(false);
+  const {deleteCollectedData} = useBeaconsState();
+
+  const handleDeleteProfile = async () => {
+    try {
+      const deleteOK = await deleteProfile();
+      if (deleteOK) {
+        await deleteCollectedData();
+        await signOut();
+      } else {
+        setDeleteError(true);
+      }
+    } catch (error) {
+      setDeleteError(true);
```

Comment:
I don't think we can set delete error when delete collected data fails. Especially if we delete profile first. When we have deleted profile I think we are past the point of no return, and need to sign out the user.

Reply:
I made a fix to this as in the same way it was done in `deleteProfile()`